src/topic/components/Participants.js

```diff
@@ -1,17 +1,44 @@
 import React from 'react';
 import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import PieChart from 'react-svg-piechart';
+import Colors from '../../styles/Colors';
+
+const Participants = props => {
+    const VOTES_NEEDED = 40;
```

Comment:
as this is a render method, you are re-declaring const meant in each rendering. What do you think to move out the VOTES_NEEDED and Chart Data out of this function?